instrumentation/action_pack/lib/opentelemetry/instrumentation/action_pack/handlers/action_controller.rb

```diff
@@ -37,21 +37,18 @@ def start(_name, _id, payload)
             attributes[OpenTelemetry::SemanticConventions::Trace::HTTP_TARGET] = request.filtered_path if request.filtered_path != request.fullpath
 
             span = OpenTelemetry::Instrumentation::Rack.current_span
-            span_name = if @span_naming == :semconv
-                          if http_route
+            if @span_naming == :semconv
```

Comment:
Should we skip this entire logic unless `span.recording?`?

Reply:
Yeah, that's a good improvement to make. Pushed up that change.